module/x/gravity/abci.go

```diff
@@ -93,7 +93,6 @@ func slashing(ctx sdk.Context, k keeper.Keeper) {
 	// Slash validator for not confirming valset requests, batch requests, logic call requests
 	valsetSlashing(ctx, k, params)
 	batchSlashing(ctx, k, params)
-	logicCallSlashing(ctx, k, params)
```

Comment:
why was this removed? what lint did it violate? It's not like it's unused code

Reply:
Duplicate code

Reply:
it's not duplicate it's similar, the lint had a false positive and this code needs to be restored, and checked over.